### ISSUE TITLE
About the number of images on KITTI used for domain adaption

### ISSUE THREAD

**Issue by SpadeLiu** [ruiliu-ai/StereoGAN#1]
Thanks for your wonderful works and codes!
I just wonder if you use the total 200 training pairs on KITTI 2015 for domain adaption, or split the 200 pairs into a training set and a validating set as those supervised methods, then use the training set for domain adaption and evaluate the performance on the validating set?

**Comment by ruiliu-ai** [comment]
We split the 200 pairs into a training set including 160 pairs and a validation set including 40 pairs.

**Comment by SpadeLiu** [comment]
> We split the 200 pairs into a training set including 160 pairs and a validation set including 40 pairs.

Could you please provide the name list of the training set and the validation set? It seems the image names in the 'txt' file are not complete.
